backend/tests/unit_tests/test_static_util.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from static.exception import FileNotExistError
+from static.util import (
+    is_image_with_specific_id_exist,
+    delete_image,
+    put_image,
+)
+
+if TYPE_CHECKING:
+    from flask import Flask
+
+_some_image_base64_content = "data:image/png;base64,somecontent"
+_some_image_uuid = "c11d5bcf-f529-4318-904d-4bc8b8d7f68a"
+
+
+class TestStorageImage:
+    def test_put_image_by_absent_uuid_should_create_image(self, app: Flask) -> None:
+        with app.app_context():
+            put_image(_some_image_base64_content, _some_image_uuid)  # Create image
+
+            assert is_image_with_specific_id_exist(_some_image_uuid)
+
+            delete_image(_some_image_uuid)  # Teardown
```

Comment:
Handle tear-down with fixture.
Also I suggest you use the [tempfile](https://docs.python.org/zh-tw/3/library/tempfile.html) module to generate random directory since you can't make sure there isn't any pre-existing file inside the `/tmp` directory.